roles/lib_openshift/src/class/oc_project.py

```diff
@@ -0,0 +1,183 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCProject(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'namespace'
```

Comment:
Is there a reason to override kind to namespace here instead of project? 

This will break the ability for non-admin users to manage their project using this module, but that would also require adding support to selectively do `oc new-project` vs `oc adm new-project`